### ISSUE TITLE
No tests found in file

### ISSUE THREAD

**Issue by Gonzo102** [machine/machine.specifications.runner.resharper#13]
I am running VS2013 update 4, Resharper 9.0, and Mspec 0.9.1.0.  The runner is not recognizing test and when I say run all test in the test project it says No Test found in file.  I have re-installed Resharper and the extension several times and re-started VS several times to no avail.  Any help would be appreciated.

Thanks Ken.

**Comment by danielmarbach** [comment]
Ping me on skype. Need to have a look. Can't guess blindly unfortunately

**Comment by Gonzo102** [comment]
Daniel,

The company I work for has Lync, but they have the skype feature turned off.  Is there another way that we can do this?  Like an online meeting or something?  In the mean time, I will see if they will let me install skype or have a way for to share a desktop.





Thanks!
Kenneth S. Hines

**Comment by danielmarbach** [comment]
Send me an email to Daniel dot Marbach at openplace dot net and I’ll send you a zoom invitation.

**Comment by Gonzo102** [comment]
Daniel,

Just checking to see If you got my email at openplace?





Thanks!
Kenneth S. Hines

**Comment by danielmarbach** [comment]
I just verified on my machine with latest resharper. It works as expected. Even pulled in Moq which introduces a conflict with the `It`. No trying to install dotCover

**Comment by Gonzo102** [comment]
Thanks.  I searched for other version of mspec and didn’t find any.  I un-installed the extension and resharper again and deleted all folders under appdata, then re-installed.  It still doesn’t work.  I also opened a case with jetbrains to see if they may know something.  I wish there was a way to tell if resharper is picking up the extension.  Do you know where  the extensions installed?





Thanks!
Kenneth S. Hines

**Comment by agross** [comment]
You could check Tools | Options | Code Inspection | Code Annotations and
see if M.S.Annotations is listed.

Beste Grüße,

Alex
-- 
Alexander Groß
http://therightstuff.de/

**Comment by Gonzo102** [comment]
Thanks.  Yes it was there.





Thanks!
Kenneth S. Hines

**Comment by Gonzo102** [comment]
Hello,

It appears I got this working.  I ran this again “ReSharperAndToolsPacked01Update.exe” and re-installed dotCover and did a repair on ReSharper and my Mspec tests are now working in both dotCover and ReSharper. The ReSharper repair option also solved a different Mspec issue that another developer was having where he could run the menu option “Run all test in solution”, but was having an issue running individual test.

I will let you know if I have any other issues.






Thanks!
Kenneth S. Hines
